app/src/main/java/org/mozilla/fenix/tabtray/TabTrayViewHolder.kt

```diff
@@ -72,7 +73,10 @@ class TabTrayViewHolder(
         if (tab.thumbnail != null) {
             thumbnailView.setImageBitmap(tab.thumbnail)
         } else {
-            imageLoader.loadIntoView(thumbnailView, ImageRequest(tab.id))
+            // Make sure we have the view's dimensions so we can load the image at the correct size
```

Comment:
This seem like a merge of another commit?
Probably will not appear anymore after a rebase.